src/components/MapDetail/components/Charts/Charts.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Grid } from 'grommet'
+import Chart from './Chart'
+
+export default function Charts() {
+  return (
+    <Grid
+      areas={[
+        { name: 'Graph1', start: [0, 0], end: [0, 0] },
+        { name: 'Graph2', start: [0, 1], end: [0, 1] },
+        { name: 'Graph3', start: [1, 0], end: [1, 0] },
+        { name: 'Graph4', start: [1, 1], end: [1, 1] },
+      ]}
+      columns={['auto', 'auto']}
+      gap='xsmall'
+      margin={{ bottom: 'xxsmall' }}
+      rows={['auto', 'auto']}
+    >
+      <Chart gridArea="Graph1" text='Kelp Over Time' />
+      <Chart gridArea="Graph2" text='Wind Patterns' />
+      <Chart gridArea="Graph3" text='Temperature' />
+      <Chart gridArea="Graph4" text='Wave Patterns' />
```

Comment:
This is my first dip into Grommet's Grid, the following is more for my understanding than issues that need to be addressed. Will there always be four Graphs? I'm not sure if I'm interpreting correctly, but would this always be a 4 by 4 grid, regardless of container size? Or do we want it to change dynamically, maybe for small or xsmall screen sizes?

Reply:
Good question. Per the design, there is a 4x4 grid, but we haven't discussed responsiveness yet. If that section does end up wrapping and stacking the charts in a column, we probably shouldn't use a grid here because it doesn't allow wrapping very easily.

Reply:
You can use the responsive context to adjust the props of the Grid accordingly.